client/src/components/PageLayout/HeaderMenu.tsx

```diff
@@ -0,0 +1,112 @@
+import {
+  Button,
+  Flex,
+  Menu,
+  MenuList,
+  MenuItem,
+  MenuButton,
+} from '@chakra-ui/react';
+import React from 'react';
+import NextLink from 'next/link';
+import styles from '../../styles/Header.module.css';
+import { useLogin } from 'hooks/useAuth';
+const serverUrl = process.env.NEXT_PUBLIC_SERVER_URL || 'http://localhost:5000';
+
+// TODO: distinguish between logging into the app and logging into Auth0. Maybe
+// use sign-in for the app?
+const LoginButton = () => {
+  const login = useLogin();
+  return (
+    <MenuItem
+      onClick={login}
+      fontWeight="600"
+      background={'gray.85'}
+      color={'gray.10'}
+      height={'100%'}
+      borderRadius={'5px'}
+      _hover={{ color: 'gray.85' }}
+    >
+      Log In
+    </MenuItem>
+  );
+};
```

Comment:
I think it's better to move this back to header

- we can create a component later for login/logout or similar function like leave and join chapter